doc/ref/debug.xml

```diff
@@ -1007,8 +1007,9 @@ brk> GetRecursionDepth();
 0
 brk> return;
 gap> SetRecursionTrapInterval(-1);
-SetRecursionTrapInterval( <interval> ): <interval> must be a non-negative smal\
-l integer
+Error, SetRecursionTrapInterval: <interval> must be a small integer greater \
+than 5 (not the integer -1)
+an 5 (not the value 'fail')
```

Comment:
what's happening here?

Reply:
That was wrong (and I even noticed and fixed it yesterday, but then failed to force push *sigh*).